systemd/crc-routes-controller.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -x
+
+# exit early if system-mode networking
+ping -c1 gateway > /dev/null 2>&1
+[ $? -eq 2 ] && exit 0
+
+export KUBECONFIG=/opt/kubeconfig
+oc apply -f /opt/crc/routes-controller.yaml
```

Comment:
even the service start after kubelet what make sure it is serving to api call? so may be first we wait till it start serving?

Reply:
yes, need to add some kind of check to see if the API is up